### ISSUE TITLE
[Feature]: per dataset configuration file to skip/ignore false positives

### ISSUE THREAD

**Issue by yarikoptic** [NeurodataWithoutBorders/nwbinspector#117]
### What would you like to see added to the NWBInspector?

Originally requested in #37 . I think that the implementation in #86 went on a different "tangent" from what my bids-validator-rotten mind had in mind: e.g. given the "stock" (provided by nwbinspector) `dandi` configuration I want to disable some tests/checks, e.g. reacting to some benign `check_data_orientation` as given in the example of https://github.com/dandi/handbook/pull/40 .

Then I would have created some `.nwbinspector.cfg` or alike which would have instructed to ignore such errors if nwbinspector was ran in this folder. 

Also #86 has `TODO: expose to command line` -- was that done?

### Do you have any interest in helping implement the feature?

No.

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/NeurodataWithoutBorders/nwbinspector/blob/dev/.github/CODE_OF_CONDUCT.rst)
- [X] Have you ensured this feature was not already [requested](https://github.com/neurodatawithoutborders/nwbinspector/issues)?

**Comment by CodyCBakerPhD** [comment]
> Also https://github.com/NeurodataWithoutBorders/nwbinspector/pull/86 has TODO: expose to command line -- was that done?

Yes, and an especially simple one at that: `nwbinspector -c dandi`


> I want to disable some tests/checks, e.g. reacting to some benign check_data_orientation as given in the example of https://github.com/dandi/handbook/pull/40 .

You can SKIP checks in the config, sorry we don't have more documentation on that yet (hoping soon; at the very least here is a [test demonstration](https://github.com/NeurodataWithoutBorders/nwbinspector/blob/dev/tests/test_check_configuration.py#L57-L61)), but it may be easier to have a SELECT functionality, which we support at the library/CLI level but not the config level.
